src/main/java/frc/robot/constants/RobotConstants.java

```diff
@@ -99,4 +100,17 @@
         public static final int XBOX_PORT = 0;
         public static final double RUMBLE_INTERMISSION_TIME = 0.15;
     }
+
+    public static class IntakeConstants {
+        public static final TrigonTalonSRX MOTOR = IntakeComponents.MOTOR;
+    }
+
+    public static class IntakeOpenerConstants {
+        public static final TrigonTalonSRX MOTOR = IntakeOpenerComponents.MOTOR;
+        public static final double OPERATING_POWER = 0.4;
+        public static double OPENED_ANGLE = 13;
+        public static double CLOSED_ANGLE = 110;
+        public static double STALL_CURRENT_LIMIT = 20;
```

Comment:
this value is currently not used in intake opener.
if we have an encoder it is not necessary but it might be a good safety feature, you may add a boolean function: inStalled to intakeOpenerSS that checks if the motor.STALL_CURRENT >= STALL_CURRENT_LIMIT;